python/humbug/report.py

```diff
@@ -74,6 +92,23 @@ def __init__(
 
         self.is_excepthook_set = False
 
+    def create_report(
+        self,
+        token: Union[str, uuid.UUID],
+        title: str,
+        content: str,
+        tags: List[str] = [],
+        **kwargs,
+    ):
+        method = "POST"
+        report_path = f"humbug/reports"
+        json = {"title": title, "content": content, "tags": tags}
+        headers = {
+            "Authorization": f"Bearer {token}",
+        }
+        url = f"{self.url.rstrip('/')}/{report_path.rstrip('/')}"
+        make_request(method, url=url, headers=headers, json=json, **kwargs)
```

Comment:
Do request directly in this function?

Reply:
If you want to keep `make_request`, it would be because we may add other endpoints for the `/humbug` API.

In this case, the right pattern is to add an enum of valid `/humbug` endpoints and pass the endpoint as an argument to `make_request`.

My preferences is that you just use `requests.post` directly here. If/when we add other `/humbug` endpoints, we will create new methods of `Reporter`.

Reply:
done